notebooks/05-operations/02-druid-logs-asynch.ipynb

```diff
@@ -0,0 +1,381 @@
+    "  --header 'Content-Type: application/json' \\\n",
+    "  --data-raw '{\n",
+    "    \"query\": \"INSERT INTO \\\"example-wikipedia-logs\\\"\\nSELECT\\n  TIME_PARSE(\\\"timestamp\\\") AS __time,\\n  *\\nFROM TABLE(\\n  EXTERN(\\n    '\\''{\\\"type\\\": \\\"http\\\", \\\"uris\\\": [\\\"https://druid.apache.org/data/wikipedia.json.gz\\\"]}'\\'',\\n    '\\''{\\\"type\\\": \\\"json\\\"}'\\'',\\n    '\\''[{\\\"name\\\": \\\"added\\\", \\\"type\\\": \\\"long\\\"}, {\\\"name\\\": \\\"channel\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"cityName\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"comment\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"commentLength\\\", \\\"type\\\": \\\"long\\\"}, {\\\"name\\\": \\\"countryIsoCode\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"countryName\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"deleted\\\", \\\"type\\\": \\\"long\\\"}, {\\\"name\\\": \\\"delta\\\", \\\"type\\\": \\\"long\\\"}, {\\\"name\\\": \\\"deltaBucket\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"diffUrl\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"flags\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"isAnonymous\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"isMinor\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"isNew\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"isRobot\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"isUnpatrolled\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"metroCode\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"namespace\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"page\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"regionIsoCode\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"regionName\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"timestamp\\\", \\\"type\\\": \\\"string\\\"}, {\\\"name\\\": \\\"user\\\", \\\"type\\\": \\\"string\\\"}]'\\''\\n  )\\n)\\nPARTITIONED BY DAY\"\n",
+    "  }'\n",
+    "```\n",
+    "\n",
+    "Open Coordinator and Overlog combined log in a separate panel by pressing `b` and select the `coordinator-overlord` option.\n",
+    "\n",
+    "Repeat pressing `b` until you find your line marker and notice the execution history is logged in detail:\n",
+    "\n",
+    "* Tasks are queued up, including with detail written to the Metadata Database\n",
+    "* Tasks start `RUNNING`\n",
+    "* Segments identifiers are allocated and then created\n",
+    "* Segments are completed and the shutdown is recognised\n",
+    "* Segments are published\n",
+    "\n",
+    "Press `q` to exit the panel, and then hit `b` and open the Middle Manager logs.\n",
+    "\n",
+    "> Use `Y` in the `multitail` window to switch line wrapping on and off.\n",
+    "\n",
+    "In the `middleManager` log, notice:\n",
+    "\n",
+    "* The creation of Task Runners, each with their own Java configurations.\n",
```

Comment:
Perhaps add examples of the log lines that the user is supposed to be looking at... something like: 
"look for ForkingTaskRunner - Running command" - each of these is a task starting